courses/machine_learning/production_ml/3_streaming_data.ipynb

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
<p>I think PROJECT is unnecessary. </p><p> </p><blockquote>bq = bigquery. Client() </blockquote><p> </p><p>should work</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/612/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Ok. Done. Thnx.</p>